rspace/src/main/scala/coop/rchain/rspace/LMDBStore.scala

```diff
@@ -295,6 +327,12 @@ class LMDBStore[F[_], C, P, A, K] private[rspace] (
 
 object LMDBStore {
 
+  def roundTrip[K: Codec](k: K): K =
```

Comment:
There's a similar implementation in `InMemoryStore`. Let's move to a common location and re-use